### DISCUSSION TITLE
why is gpu temprature limit not available in options

### DISCUSSION THREAD

**Discussion by Y14C** [seerge/g-helper#3216] [General]
i am currently on ASUS rog strix 15 model-G513IC withRTX3050 4gb gpu i am trying to get gpu temprature limit but am not able to do so. please help

**Comment by seerge** [comment]
@yash14chouhan most likely your device does not support it. If you didn't have it in Armoury Crate - you won't have it here.

**Reply by Y14C** [reply]
thanks for the reply, still awesome app thanks a lot for making it
